helm/values.yaml

```diff
@@ -6,6 +6,7 @@
 # Deployment and Service
 ###########
 logLevel: INFO
+metricsAddress: "0.0.0.0:8888"
```

Comment:
Does this need to be exposed as a separate container/service port?

Reply:
Following is the usage:
```
Hypertrace Collector

Usage:
  collector [flags]

Flags:
      --add-instance-id             Flag to control the addition of 'service.instance.id' to the collector metrics. (default true)
      --config string               Path to the config file
  -h, --help                        help for collector
      --log-format string           Format of logs to use (json, console) (default "console")
      --log-level string            Output level of logs (DEBUG, INFO, WARN, ERROR, DPANIC, PANIC, FATAL) (default "INFO")
      --log-profile string          Logging profile to use (dev, prod)
      --mem-ballast-size-mib uint   Flag to specify size of memory (MiB) ballast to set. Ballast is not used when this is not specified. default settings: 0
      --metrics-addr string         [address]:port for exposing collector telemetry. (default "localhost:8888")
      --metrics-level Level         Output level of telemetry metrics (none, basic, normal, detailed) (default basic)
      --metrics-prefix string       Prefix to the metrics generated by the collector. (default "otelcol")
      --set stringArray             Set arbitrary component config property. The component has to be defined in the config file and the flag has a higher precedence. Array config properties are overridden and maps are joined, note that only a single (first) array property can be set e.g. -set=processors.attributes.actions.key=some_key. Example --set=processors.batch.timeout=2s
```

Reply:
That makes sense at the process level, but does the container/service still need to expose that port to support scraping?